zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/group/PersistedGroup.java

```diff
@@ -41,4 +42,15 @@ public PersistedGroup setName(final String name) {
     nameProp.setValue(name);
     return this;
   }
+
+  public PersistedGroup copy() {
```

Comment:
❓ Is this method used somewhere ?

Reply:
Not really, I was following the `Persisted*` classes pattern. But you make a good point, I'll remove it. It can be added once it is needed.